src/components/ProjectDetailPage/__tests__/ProjectDetailPage.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react';
+import { ProjectDetailPage } from '../ProjectDetailPage';
+import { MemoryRouter, Route } from 'react-router-dom';
+
+test('renders ProjectDetailPage', () => {
+  render(
+    <MemoryRouter initialEntries={['projects/101']}>
+      <Route path="projects/:id">
+        <ProjectDetailPage />
+      </Route>
+    </MemoryRouter>
+  );
+});
```

Comment:
Hi @jakubvanko ,you can refer to Unit Test for ProjectListPage to have similar test for your page, see https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/components/ProjectsList/__tests__/ProjectsList.test.tsx
For example, you can test for "rendering ProjectDetails page to have the right data" and also have a snopshot test like that proceeded in ProjectsList.test.tsx.

Reply:
e.g, to verify some data exist on the page ma looks like this:
`    const firstProject = screen.getByText(projectsMock[0].name);
    expect(firstProject).toBeInTheDocument();`

and code for verify the snapshot can be found here: https://github.com/project-ncl/pnc-web-ui-react/blob/f5e16df28ff5ddfe8105dbcf5236349cdd8be008/src/components/ProjectsList/__tests__/ProjectsList.test.tsx#L33-L40